sigar_linux.go

```diff
@@ -257,6 +257,10 @@ func (self *FileSystemList) Get() error {
 
 	err := readFile("/etc/mtab", func(line string) bool {
 		fields := strings.Fields(line)
+		// Ignore the fields not correct
+		if len(fields) <= 3 {
```

Comment:
Well, II would write `if len(fields) < 4` because you expect at least 4 fields. Thus, `3` is not a relevant number here.

Could you also create a constant for this minimum fields count, so that we avoid having a magic number?

Plus, the comment should be rephrased "Ignore invalid lines with too few fields".

Reply:
@jannson Could you please take a look at the above?